app/src/main/java/com/github/studydistractor/sdp/ProcrastinationActivity.kt

```diff
@@ -0,0 +1,33 @@
+package com.github.studydistractor.sdp
+
+import android.os.Parcel
+import android.os.Parcelable
+
+data class ProcrastinationActivity(
+    val name: String? = null,
+    val description: String? = null) :
+    Parcelable {

```

Comment:
This should not be used in future sprints. We want to manage state with compose (see [this](https://developer.android.com/jetpack/compose/state))

Reply:
Converted to issue : #36